realm/realm-library/src/androidTest/java/io/realm/RealmAsyncQueryTests.java

```diff
@@ -127,23 +121,76 @@ public void onSuccess() {
 
     @Test
     @RunTestInLooperThread
+    public void executeTransactionAsync_onSuccessCallerRealmClosed() throws Throwable {
```

Comment:
Yup ... we swallowed it before as well as the `onError`. But maybe the old behavior is correct since we have a `RealmNotifier.close()` which is supposed to remove all callbacks and stop message deliver when the Realm closed. I think it will be more correct to just swallow the callback as before. What do you think? @Zhuinden @cmelchior